log/log.go

```diff
@@ -14,12 +13,9 @@ import (
 func Init(conf *Config) {
 	// This mostly should be the responsibility of the app itself but hey
 	zerolog.SetGlobalLevel(conf.Level)
-	zerolog.TimeFieldFormat = zerolog.TimeFormatUnix
-	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr})
-
-	zerolog.TimestampFunc = func() time.Time {
-		return time.Now().In(time.Local)
-	}
```

Comment:
we can add this back in... it felt like it was conflicting with TimeFieldFormat.